airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/templates/render.py

```diff
@@ -72,10 +72,113 @@ def dataframe_to_table_html(df: pd.DataFrame, column_mapping: List[ColumnInfo])
     )
 
 
+def value_to_emoji(value: Any) -> str:
+    if value is True:
+        return "✅"
+    elif value is False:
+        return "❌"
+    elif value is None or pd.isna(value):
+        return "❓"
+    else:
+        return str(value)
```

Comment:
nit: use a mapping:
```python
value_to_emoji = {True: "✅", False: ,"❌", None: "❓"}
return value_to_emoji.get(value, str(value))
```

Reply:
I like this! but unfortunately we cant use Bools as keys 😢 , We could stringify the value before getting but then the None case gets strange.